resources/entrypoint.sh

```diff
@@ -17,7 +19,14 @@ if [ "x$SNMP_TRAP_DEST" != "x" ]; then
     sed -i -E "s/^(Trapsink ).* (.*)\$/\1 $SNMP_TRAP_DEST \2/g" /etc/snmp/snmpd.conf
 fi
 
-/etc/init.d/snmpd start
-srvadmin-services.sh restart
+# Run any passed commands
+if [ "$#" -gt 0 ]; then
+    # Start Dell services
+    echo "Starting Dell services, this may take a few minutes..."
+    systemctl start snmpd.service
+    srvadmin-services.sh start
```

Comment:
I notice that `srvadmin-services.sh start` is only run in the entrypoint script if `CMD` is set. Probably not an issue since a default `CMD` is set in the Dockerfile, but for clarity would it make sense to move the service startup outside of the `CMD` eval block?

Reply:
Good point - I had considered this, then I tried to see if it was possible not to end up in that block, but it seemed impossible.  I think I'll the "start services" portion out and just leave the eval statement in there.